src/praisonai-agents/praisonaiagents/session/sqlite_store.py

```
@@ -0,0 +1,387 @@
+    def add_message(
+        self,
+        session_id: str,
+        role: str,
+        content: str,
+        metadata: Optional[Dict[str, Any]] = None,
+    ) -> bool:
+        ok = super().add_message(session_id, role, content, metadata)
+        if ok:
+            try:
+                self._index_session(self._read_session_fresh(session_id))
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)